### DISCUSSION TITLE
tootctl self-destruct timeout error

### DISCUSSION THREAD

**Discussion by Lisdexamfetimon** [mastodon/mastodon#31028] [General]
This error happens when I self-destruct the server, also when retrying:

/home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis/connection/hiredis.rb:60:in `rescue in read': Connection timed out (Redis::TimeoutError)
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis/connection/hiredis.rb:55:in `read'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis/client.rb:311:in `block in read'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis/client.rb:299:in `io'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis/client.rb:310:in `read'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis/client.rb:238:in `block (2 levels) in call_pipelined'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis/client.rb:234:in `each'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis/client.rb:234:in `each_with_index'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis/client.rb:234:in `block in call_pipelined'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis/client.rb:279:in `block (2 levels) in process'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis/client.rb:411:in `ensure_connected'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis/client.rb:269:in `block in process'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis/client.rb:356:in `logging'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis/client.rb:268:in `process'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis/client.rb:233:in `call_pipelined'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis/client.rb:200:in `block in call_pipeline'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis/client.rb:344:in `with_reconnect'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis/client.rb:198:in `call_pipeline'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis.rb:177:in `block in pipelined'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis.rb:265:in `block in synchronize'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis.rb:265:in `synchronize'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis.rb:265:in `synchronize'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis.rb:171:in `pipelined'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/sidekiq-6.5.12/lib/sidekiq/client.rb:201:in `block in raw_push'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/connection_pool-2.4.1/lib/connection_pool.rb:110:in `block (2 levels) in with'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/connection_pool-2.4.1/lib/connection_pool.rb:109:in `handle_interrupt'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/connection_pool-2.4.1/lib/connection_pool.rb:109:in `block in with'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/connection_pool-2.4.1/lib/connection_pool.rb:106:in `handle_interrupt'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/connection_pool-2.4.1/lib/connection_pool.rb:106:in `with'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/sidekiq-6.5.12/lib/sidekiq/client.rb:198:in `raw_push'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/sidekiq-6.5.12/lib/sidekiq/client.rb:120:in `push_bulk'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/sidekiq-6.5.12/lib/sidekiq/client.rb:151:in `push_bulk'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/sidekiq-bulk-0.2.0/lib/sidekiq/bulk.rb:19:in `push_bulk!'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/sidekiq-bulk-0.2.0/lib/sidekiq/bulk.rb:6:in `block in push_bulk'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/sidekiq-bulk-0.2.0/lib/sidekiq/bulk.rb:5:in `each'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/sidekiq-bulk-0.2.0/lib/sidekiq/bulk.rb:5:in `each_slice'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/sidekiq-bulk-0.2.0/lib/sidekiq/bulk.rb:5:in `each'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/sidekiq-bulk-0.2.0/lib/sidekiq/bulk.rb:5:in `map'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/sidekiq-bulk-0.2.0/lib/sidekiq/bulk.rb:5:in `push_bulk'
        from /home/mastodon/live/lib/mastodon/cli/main.rb:129:in `block in self_destruct'
        from /home/mastodon/live/lib/mastodon/cli/main.rb:140:in `block in self_destruct'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/activerecord-7.0.8.4/lib/active_record/relation/batches.rb:71:in `each'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/activerecord-7.0.8.4/lib/active_record/relation/batches.rb:71:in `block in find_each'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/activerecord-7.0.8.4/lib/active_record/relation/batches.rb:138:in `block in find_in_batches'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/activerecord-7.0.8.4/lib/active_record/relation/batches.rb:245:in `block in in_batches'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/activerecord-7.0.8.4/lib/active_record/relation/batches.rb:229:in `loop'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/activerecord-7.0.8.4/lib/active_record/relation/batches.rb:229:in `in_batches'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/activerecord-7.0.8.4/lib/active_record/relation/batches.rb:137:in `find_in_batches'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/activerecord-7.0.8.4/lib/active_record/relation/batches.rb:70:in `find_each'
        from /home/mastodon/live/lib/mastodon/cli/main.rb:140:in `self_destruct'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/thor-1.3.1/lib/thor/command.rb:28:in `run'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/thor-1.3.1/lib/thor/invocation.rb:127:in `invoke_command'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/thor-1.3.1/lib/thor.rb:527:in `dispatch'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/thor-1.3.1/lib/thor/base.rb:584:in `start'
        from bin/tootctl:9:in `block in <main>'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/chewy-7.3.4/lib/chewy/strategy.rb:60:in `wrap'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/chewy-7.3.4/lib/chewy.rb:154:in `strategy'
        from bin/tootctl:8:in `<main>'
/home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis/connection/hiredis.rb:60:in `rescue in read': Redis::TimeoutError (Redis::TimeoutError)
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis/connection/hiredis.rb:55:in `read'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis/client.rb:311:in `block in read'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis/client.rb:299:in `io'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis/client.rb:310:in `read'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis/client.rb:238:in `block (2 levels) in call_pipelined'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis/client.rb:234:in `each'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis/client.rb:234:in `each_with_index'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis/client.rb:234:in `block in call_pipelined'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis/client.rb:279:in `block (2 levels) in process'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis/client.rb:411:in `ensure_connected'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis/client.rb:269:in `block in process'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis/client.rb:356:in `logging'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis/client.rb:268:in `process'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis/client.rb:233:in `call_pipelined'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis/client.rb:200:in `block in call_pipeline'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis/client.rb:344:in `with_reconnect'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis/client.rb:198:in `call_pipeline'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis.rb:177:in `block in pipelined'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis.rb:265:in `block in synchronize'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis.rb:265:in `synchronize'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis.rb:265:in `synchronize'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis.rb:171:in `pipelined'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/sidekiq-6.5.12/lib/sidekiq/client.rb:201:in `block in raw_push'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/connection_pool-2.4.1/lib/connection_pool.rb:110:in `block (2 levels) in with'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/connection_pool-2.4.1/lib/connection_pool.rb:109:in `handle_interrupt'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/connection_pool-2.4.1/lib/connection_pool.rb:109:in `block in with'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/connection_pool-2.4.1/lib/connection_pool.rb:106:in `handle_interrupt'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/connection_pool-2.4.1/lib/connection_pool.rb:106:in `with'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/sidekiq-6.5.12/lib/sidekiq/client.rb:198:in `raw_push'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/sidekiq-6.5.12/lib/sidekiq/client.rb:120:in `push_bulk'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/sidekiq-6.5.12/lib/sidekiq/client.rb:151:in `push_bulk'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/sidekiq-bulk-0.2.0/lib/sidekiq/bulk.rb:19:in `push_bulk!'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/sidekiq-bulk-0.2.0/lib/sidekiq/bulk.rb:6:in `block in push_bulk'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/sidekiq-bulk-0.2.0/lib/sidekiq/bulk.rb:5:in `each'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/sidekiq-bulk-0.2.0/lib/sidekiq/bulk.rb:5:in `each_slice'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/sidekiq-bulk-0.2.0/lib/sidekiq/bulk.rb:5:in `each'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/sidekiq-bulk-0.2.0/lib/sidekiq/bulk.rb:5:in `map'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/sidekiq-bulk-0.2.0/lib/sidekiq/bulk.rb:5:in `push_bulk'
        from /home/mastodon/live/lib/mastodon/cli/main.rb:129:in `block in self_destruct'
        from /home/mastodon/live/lib/mastodon/cli/main.rb:140:in `block in self_destruct'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/activerecord-7.0.8.4/lib/active_record/relation/batches.rb:71:in `each'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/activerecord-7.0.8.4/lib/active_record/relation/batches.rb:71:in `block in find_each'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/activerecord-7.0.8.4/lib/active_record/relation/batches.rb:138:in `block in find_in_batches'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/activerecord-7.0.8.4/lib/active_record/relation/batches.rb:245:in `block in in_batches'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/activerecord-7.0.8.4/lib/active_record/relation/batches.rb:229:in `loop'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/activerecord-7.0.8.4/lib/active_record/relation/batches.rb:229:in `in_batches'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/activerecord-7.0.8.4/lib/active_record/relation/batches.rb:137:in `find_in_batches'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/activerecord-7.0.8.4/lib/active_record/relation/batches.rb:70:in `find_each'
        from /home/mastodon/live/lib/mastodon/cli/main.rb:140:in `self_destruct'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/thor-1.3.1/lib/thor/command.rb:28:in `run'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/thor-1.3.1/lib/thor/invocation.rb:127:in `invoke_command'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/thor-1.3.1/lib/thor.rb:527:in `dispatch'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/thor-1.3.1/lib/thor/base.rb:584:in `start'
        from bin/tootctl:9:in `block in <main>'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/chewy-7.3.4/lib/chewy/strategy.rb:60:in `wrap'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/chewy-7.3.4/lib/chewy.rb:154:in `strategy'
        from bin/tootctl:8:in `<main>'
/home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/hiredis-0.6.3/lib/hiredis/ext/connection.rb:19:in `read': Resource temporarily unavailable (Errno::EAGAIN)
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/hiredis-0.6.3/lib/hiredis/ext/connection.rb:19:in `read'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis/connection/hiredis.rb:56:in `read'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis/client.rb:311:in `block in read'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis/client.rb:299:in `io'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis/client.rb:310:in `read'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis/client.rb:238:in `block (2 levels) in call_pipelined'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis/client.rb:234:in `each'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis/client.rb:234:in `each_with_index'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis/client.rb:234:in `block in call_pipelined'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis/client.rb:279:in `block (2 levels) in process'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis/client.rb:411:in `ensure_connected'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis/client.rb:269:in `block in process'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis/client.rb:356:in `logging'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis/client.rb:268:in `process'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis/client.rb:233:in `call_pipelined'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis/client.rb:200:in `block in call_pipeline'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis/client.rb:344:in `with_reconnect'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis/client.rb:198:in `call_pipeline'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis.rb:177:in `block in pipelined'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis.rb:265:in `block in synchronize'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis.rb:265:in `synchronize'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis.rb:265:in `synchronize'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/redis-4.8.1/lib/redis.rb:171:in `pipelined'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/sidekiq-6.5.12/lib/sidekiq/client.rb:201:in `block in raw_push'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/connection_pool-2.4.1/lib/connection_pool.rb:110:in `block (2 levels) in with'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/connection_pool-2.4.1/lib/connection_pool.rb:109:in `handle_interrupt'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/connection_pool-2.4.1/lib/connection_pool.rb:109:in `block in with'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/connection_pool-2.4.1/lib/connection_pool.rb:106:in `handle_interrupt'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/connection_pool-2.4.1/lib/connection_pool.rb:106:in `with'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/sidekiq-6.5.12/lib/sidekiq/client.rb:198:in `raw_push'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/sidekiq-6.5.12/lib/sidekiq/client.rb:120:in `push_bulk'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/sidekiq-6.5.12/lib/sidekiq/client.rb:151:in `push_bulk'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/sidekiq-bulk-0.2.0/lib/sidekiq/bulk.rb:19:in `push_bulk!'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/sidekiq-bulk-0.2.0/lib/sidekiq/bulk.rb:6:in `block in push_bulk'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/sidekiq-bulk-0.2.0/lib/sidekiq/bulk.rb:5:in `each'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/sidekiq-bulk-0.2.0/lib/sidekiq/bulk.rb:5:in `each_slice'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/sidekiq-bulk-0.2.0/lib/sidekiq/bulk.rb:5:in `each'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/sidekiq-bulk-0.2.0/lib/sidekiq/bulk.rb:5:in `map'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/sidekiq-bulk-0.2.0/lib/sidekiq/bulk.rb:5:in `push_bulk'
        from /home/mastodon/live/lib/mastodon/cli/main.rb:129:in `block in self_destruct'
        from /home/mastodon/live/lib/mastodon/cli/main.rb:140:in `block in self_destruct'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/activerecord-7.0.8.4/lib/active_record/relation/batches.rb:71:in `each'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/activerecord-7.0.8.4/lib/active_record/relation/batches.rb:71:in `block in find_each'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/activerecord-7.0.8.4/lib/active_record/relation/batches.rb:138:in `block in find_in_batches'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/activerecord-7.0.8.4/lib/active_record/relation/batches.rb:245:in `block in in_batches'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/activerecord-7.0.8.4/lib/active_record/relation/batches.rb:229:in `loop'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/activerecord-7.0.8.4/lib/active_record/relation/batches.rb:229:in `in_batches'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/activerecord-7.0.8.4/lib/active_record/relation/batches.rb:137:in `find_in_batches'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/activerecord-7.0.8.4/lib/active_record/relation/batches.rb:70:in `find_each'
        from /home/mastodon/live/lib/mastodon/cli/main.rb:140:in `self_destruct'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/thor-1.3.1/lib/thor/command.rb:28:in `run'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/thor-1.3.1/lib/thor/invocation.rb:127:in `invoke_command'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/thor-1.3.1/lib/thor.rb:527:in `dispatch'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/thor-1.3.1/lib/thor/base.rb:584:in `start'
        from bin/tootctl:9:in `block in <main>'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/chewy-7.3.4/lib/chewy/strategy.rb:60:in `wrap'
        from /home/mastodon/live/vendor/bundle/ruby/3.2.0/gems/chewy-7.3.4/lib/chewy.rb:154:in `strategy'
        from bin/tootctl:8:in `<main>'

**Comment by mjankowski** [comment]
Did you ever sort this out? If not, can you confirm which version of mastodon this was on and/or whether you can still replicate in latest?
